### DISCUSSION TITLE
Is it possible to add Youtube Music client?

### DISCUSSION THREAD

**Discussion by Akopov4** [clementine-player/Clementine#7260] [Ideas]
Hi, Is it possible to add YouTube client in the internet tab as it is already done for Spotify?

**Comment by fitojb** [comment]
Ideally, yes. Realistically, no.

Back when Clementine was created, lots of online services had a friendly attitude towards people building third-party/unofficial desktop clients, but over the years, those services have progressively shuttered their APIs or severely limited them in efforts to monetize. That’s why, if you study Clementine’s commit history, it has seen removals of online services it once supported. And now there is no one with enough energy and time to chase down ever-evolving paths to keep supporting online services. And as an open-source project with volunteers behind it, no one is in the possibility of paying the exorbitant fees that online services now change for the use of their remaining APIs.
